libethereum/Client.cpp

```diff
@@ -55,7 +55,7 @@ void VersionChecker::setOk()
 		{
 			cwarn << "Unhandled exception! Failed to create directory: " << m_path << "\n" << boost::current_exception_diagnostic_information();
 		}
-		writeFile(m_path + "/protocol", rlp(eth::c_protocolVersion));
```

Comment:
for good practice, should be explicitly `eth::c_protocolVersion` since we now have `p2p::c_protocolVersion`.